compiler/crates/relay-compiler/src/build_project/artifact_generated_types.rs

```diff
@@ -27,8 +30,9 @@ impl ArtifactGeneratedTypes {
         operation: &OperationDefinition,
         skip_types: bool,
         is_client_only: bool,
+        project_config: &ProjectConfig,
```

Comment:
Based on the principle of least knowledge should we maybe _just_ pass the language here instead of the full project_config?

Reply:
Fixed